src/app/static/src/app/featureSwitches.ts

```diff
@@ -1,5 +1,5 @@
 const urlParams = new URLSearchParams(window.location.search);
-const stratAcrossRegions = !!urlParams.get("stratAcrossRegions");
+const stratAcrossRegions = !urlParams.get("stratAcrossRegions");
```

Comment:
do we want to totally remove the feature switch?

Reply:
Given the history of wanting to strategically hide this, let's maybe keep it in for now.